src/main/features/dashboard/views/macro/claimStatus/redeterminationRequestedClaimant.njk

```diff
@@ -0,0 +1,20 @@
+{% macro redeterminationClaimantDashboard(claim) %}
```

Comment:
`claim` param is not required

Reply:
not needed because it's loaded on every page as part of the index? I am using claim in here: { defendantName: claim.claimData.defendant.name

Reply:
As not used in this macro, so don't need to pass in.

Reply:
was looking at the wrong macro when I replied. have removed it now. thanks

Reply:
That's what I thought ;)